libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1787,6 +1797,17 @@ MAV_RESULT GCS_MAVLINK::handle_rc_bind(const mavlink_command_long_t &packet)
     return MAV_RESULT_ACCEPTED;
 }
 
+int64_t GCS_MAVLINK::timesync_timestamp_ns() const
```

Comment:
a comment explaining the purpose of this function would be nice (especially noting that it should be used only when responding to a TIMESYNC)